plugins/xrandr/msd-xrandr-manager.c

```diff
@@ -1716,12 +1716,28 @@ make_menu_item_for_output_title (MsdXrandrManager *manager, MateRROutputInfo *ou
 {
         GtkWidget       *item;
         GtkStyleContext *context;
+        GtkCssProvider  *provider;
         GtkWidget       *label;
         char *str;
 
         item = gtk_menu_item_new ();
         context = gtk_widget_get_style_context (item);
         gtk_style_context_add_class (context, "xrandr-applet");
+        provider = gtk_css_provider_new ();
+        gtk_css_provider_load_from_data (provider,
+					".mate-panel-menu-bar menuitem.xrandr-applet:disabled > label {\n"
+					"background-color: #e3ffaa;\n"
```

Comment:
Hmm, but the background color should be different for each monitor... it's obtained via `mate_rr_labeler_get_rgba_for_output` (when https://github.com/mate-desktop/mate-settings-daemon/pull/221 isn't applied).